ui/pages/asset/components/asset-page.tsx

```diff
@@ -479,7 +479,7 @@ const AssetPage = ({
               {t('yourActivity')}
             </Text>
             {type === AssetType.native ? (
-              <TransactionList hideTokenTransactions hideNetworkFilter />
+              <TransactionList hideNetworkFilter />
```

Comment:
Curious was there a reason why we wanted to hide token transactions on the asset page? 🤔